packages/cli/src/ui/components/__snapshots__/EditorSettingsDialog.test.tsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`EditorSettingsDialog > renders correctly 1`] = `
 │   2. Vim                                     These editors are currently supported. Please note  │
 │                                              that some editors cannot be used in sandbox mode.   │
 │   Apply To                                                                                       │
-│ ● 1. User Settings                           Your preferred editor is: None.                     │
+│ ● 1. User Settings                           Your preferred editor is: VS Code.                  │
```

Comment:
same here, if this is unrelated to the change, let's move it to a separate PR to keep things clean